integration-tests/src/main/java/org/apache/druid/testing/utils/KinesisAdminClient.java

```diff
@@ -129,18 +132,28 @@ public void updateShardCount(String streamName, int newShardCount, boolean block
     }
   }
 
+  @Override
   public boolean isStreamActive(String streamName)
   {
     StreamDescription streamDescription = getStreamDescription(streamName);
     return verifyStreamStatus(streamDescription, StreamStatus.ACTIVE);
   }
 
+  @Override
   public int getStreamShardCount(String streamName)
   {
     StreamDescription streamDescription = getStreamDescription(streamName);
     return getStreamShardCount(streamDescription);
   }
 
+  @Override
+  public boolean verfiyShardCountUpdated(String streamName, int oldShardCount, int newShardCount)
+  {
+    int actualShardCount = getStreamShardCount(streamName);
+    return actualShardCount == oldShardCount + newShardCount;
```

Comment:
Suggest adding a brief comment here noting that Kinesis doesn't immediately drop the old shards after the resharding which is why the counts are summed

Reply:
Done